app/protocols/theme-handler.js

```diff
@@ -0,0 +1,54 @@
+import path from 'path'
+import { fileURLToPath } from 'url'
+import fs from 'fs'
```

Comment:
Whenever you read from the file system, make sure you use [ScopedFS](https://github.com/AgregoreWeb/agregore-browser/blob/master/app/protocols/browser-protocol.js#L24) otherwise users can use malicious path names like `../../../config/yourpasswords` to extract arbitrary files from the filesystem